storage/src/adb/current.rs

```diff
@@ -195,116 +207,164 @@ impl<
     /// Updates `key` to have value `value`. If the key already has this same value, then this is a
     /// no-op. The operation is reflected in the snapshot, but will be subject to rollback until the
     /// next successful `commit`.
-    pub async fn update(
-        &mut self,
-        hasher: &mut H,
-        key: K,
-        value: V,
-    ) -> Result<UpdateResult, Error> {
-        let update_result = self.any.update(hasher, key, value).await?;
+    pub async fn update(&mut self, key: K, value: V) -> Result<UpdateResult, Error> {
```

Comment:
Love that we don't need to provide the hasher anymore 🚀

Reply:
Yup, only the non-mut methods (specifically: root()) require you provide it now.